examples/contrastive/contrastive.cpp

```diff
@@ -50,8 +50,13 @@ int main(int argc, char ** argv) {
     llama_model * model_amateur = llama_load_model_from_file(params_amateur.model.c_str(), model_params);
 
 
-    if (model_expert == NULL or model_amateur == NULL) {
```

Comment:
Fun fact: The above only works (with some compilers) because of compatibility with ISO 646, a standard first published in 1967. In C and C++ it's spelled `||`.